webcompat/form.py

```diff
@@ -147,6 +148,12 @@ def domain_name(url):
     # testing if it's an http URL
     if url.startswith(SCHEMES):
         domain = urlparse.urlparse(url).netloc
+        if domain.lower().startswith('www.'):
+            domain = domain.split('www.')[1]
```

Comment:
It doesn't give us benefit to remove the `www.`, it's a domain on its own. Plus it leads to the potential mistake of cutting off `www.TLD` example: `www.org`.